CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unrealeased]
 
+## [3.0.1] - 2025-05-15
```

Comment:
Is the Elasticsearch addition a new feature? Shouldn't it be a minor version bump (`3.1.0`)?

Reply:
@kernicPanel I also agree with @wilbrdt.